CARET_trace/test/test_trace_node.cpp

```diff
@@ -207,27 +252,49 @@ TEST_F(CaretTraceNodeTest, TestMeasureTransition)
 {
   init_context();
 
-  {  // subscribe start message: MEASURE -> PREPARE
+  {  // subscribe start message with lttng session: MEASURE -> PREPARE
     set_status(TRACE_STATUS::RECORD);
-
     EXPECT_TRUE(node_->get_status() == TRACE_STATUS::RECORD);
 
+    set_lttng_session_running_return_value(true);
     auto start_msg = get_start_msg();
     node_->start_callback(std::move(start_msg));
 
-    EXPECT_TRUE(node_->get_status() == TRACE_STATUS::PREPARE);
+    EXPECT_TRUE(node_->get_status() == TRACE_STATUS::RECORD);
   }
 
-  {  // subscribe end message: MEASURE -> WAIT
+  {  // subscribe start message without lttng session: MEASURE -> PREPARE
```

Comment:
Minor fix of comment.
```suggestion
  {  // subscribe start message without lttng session: RECORD -> RECORD
```